contracts/contracts/strategies/CurvePoolBooster.sol

```diff
@@ -0,0 +1,309 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Initializable } from "../utils/Initializable.sol";
+import { Strategizable } from "../governance/Strategizable.sol";
+import { ICampaingRemoteManager } from "../interfaces/ICampaignRemoteManager.sol";
+
+contract CurvePoolBooster is Initializable, Strategizable {
+    ////////////////////////////////////////////////////
+    /// --- CONSTANTS && IMMUTABLES
+    ////////////////////////////////////////////////////
+    uint16 public constant BASE_FEE = 10_000; // 100%
+    address public immutable gauge;
+    address public immutable rewardToken;
+    address public immutable campaignRemoteManager;
+    uint256 public immutable targetChainId;
+
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    uint16 public fee;
+    address public feeCollector;
+    uint256 public campaignId;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event FeeUpdated(uint16 _newFee);
+    event FeeCollectorUpdated(address _newFeeCollector);
+    event CampaignIdUpdated(uint256 _newId);
+    event BribeCreated(
+        address gauge,
+        address rewardToken,
+        uint256 maxRewardPerVote,
+        uint256 totalRewardAmount
+    );
+    event TotalRewardAmountUpdated(uint256 extraTotalRewardAmount);
+    event NumberOfPeriodsUpdated(uint8 extraNumberOfPeriods);
+    event RewardPerVoteUpdated(uint256 newMaxRewardPerVote);
+    event RescueTokens(address token, uint256 amount, address receiver);
```

Comment:
nit: `TokensRescued` so that it's inline with how other events are named

Reply:
Indeed! Done in this commit https://github.com/OriginProtocol/origin-dollar/pull/2327/commits/4ec38c8d849e8d5ba1ce1654d329e9fda1d501fd